src/sidebar/services/router.js

```diff
@@ -29,7 +27,13 @@ export class RouterService {
   currentRoute() {
     const path = this._window.location.pathname;
     const pathSegments = path.slice(1).split('/');
-    const params = queryString.parse(this._window.location.search);
+    const searchParams = new URLSearchParams(this._window.location.search);
+
+    /** @type {Record<string, string>} */
+    const params = {};
+    for (let [key, value] of searchParams) {
+      params[key] = value;
+    }
```

Comment:
I want to be a billion-percent sure that the intended result for a search param string, e.g. `?foo=1&foo=2&bar=3` here is:

```
{
   foo: 2,
   bar: 3
}
```

i.e. subsequent repeated key's value will supersede earlier value for the same key

Perhaps worth documenting expected behavior with a comment, if this differs from other places we use `URLSearchParams`

Reply:
p.s. I violated my own personal peeve of people using percentage values >100.

Reply:
> i.e. subsequent repeated key's value will supersede earlier value for the same key

That's correct. Currently there are no routes which make use of repeated parameters so this approach is safe. If we do add such a route in future (say we allowed the `/notebook` route to take a repeated `document_url` parameter for example) then we'd need to change this. In this case the logic would need to be aware of which route parameters should be array-valued and which should not.